docs/operate-and-deploy/installation/server-config/security.md

```diff
@@ -817,21 +796,21 @@ cluster to allow ksqlDB to operate:
 
 ```bash
 # Allow ksqlDB to discover the cluster:
-bin/kafka-acls --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --operation DescribeConfigs --cluster
+bin/kafka-acls bootstrap-server=localhost:9092 --add --allow-principal User:KSQL1 --allow-host 198.51.100.0 --allow-host 198.51.100.1 --allow-host 198.51.100.2 --operation DescribeConfigs --cluster
```

Comment:
One more minor correction: `--bootstrap-server` instead of `bootstrap-server`

Reply:
Ah of course, thanks!